packages/web-components/@sfx/base/test/base.test.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, spy, stub } from './utils';
+import Base from '../src/base';
+import { LitElement } from 'lit-element';
+import { TemplateResult } from 'lit-element';
+import * as BaseUtils from '../src/utils';
+
+
+describe('Base Class', () => {
+  let base: any = {}
+  beforeEach(() => {
+    base = new Base();
+  });
+
+  it('should extend from LitElement', () => {
+    const base = new Base();
+    expect(base).to.be.an.instanceof(LitElement);
+  });
+
+  describe('constructor', () => {
+    it('should call addSlots', () => {
+      const addSlotsStub = stub(Base.prototype, 'addSlots')
+      base = new Base();
+      expect(addSlotsStub).to.have.been.called;
+    });
+  });
+
+  describe('first update', () => {
```

Comment:
For consistency, use the method name:

```suggestion
  describe('firstUpdate', () => {
```